### DISCUSSION TITLE
The fp process cannot iterate after dpgen training is complete

### DISCUSSION THREAD

**Discussion by Baopidragon1** [deepmodeling/dpgen#1572] [Q&A]
[param1.json](https://github.com/user-attachments/files/16043297/param1.json)
I have three qustions.
1. My training set contains computational results from 300-8000K for multiple interface structures with the goal of making a reactive force field. However, at the end of the first training round of 200000 steps, rmse_f_trn only reaches the level of x.xe-01. Is this one caused by the training set?
2. After the training is completed and the model_devi process is run, the level of max_devi_f only reaches x.xe+00, at which point the number of failed structures is always 100 percent by setting the values of model_devi_f_trust_hi and model_devi_f_trust_lo, which are commonly used online. So I cranked up the model_devi_f_trust_hi and model_devi_f_trust_lo values to get the fp process to run. I was sure that the fp results were converging, but the results show no convergence. Here is a screenshot of the relevant input parameter file and error reporting. I hope some expert can help to solve the problem.
3. 
<img width="832" alt="20240630132556" src="https://github.com/deepmodeling/dpgen/assets/154771059/17cadfdd-715f-44f7-805c-86771292d0f6">
<img width="834" alt="20240630132547" src="https://github.com/deepmodeling/dpgen/assets/154771059/88bf7126-06e2-434a-a5e1-85089bc833ea">
[param1.json](https://github.com/user-attachments/files/16043298/param1.json)

**Comment by njzjz** [comment]
It will be helpful to post a file that you believe converged but the program reported not.
